main.go

```diff
@@ -337,7 +337,11 @@ func buildLogArray() error {
 			e = filepath.Walk(absLogPath, func(path string, info os.FileInfo, err error) error {
 				if err == nil && logRegExp.MatchString(info.Name()) {
 					if filepath.IsAbs(path) {
-						logs = append(logs, path)
+						if fileInfo, err := os.Stat(path); err == nil {
```

Comment:
There is no need to call stat, Walk already gives you the FileInfo

Reply:
removed unneeded call.